anaconda_linter/lint/check_completeness.py

```diff
@@ -312,6 +313,21 @@ def check_recipe(self, recipe):
             self.message(section="about", severity=WARNING)
 
 
+class documentation_specifies_language(LintCheck):
+    """Use the generic link not a language specific one
+
+    Please use PACKAGE.readthedocs.io not  PACKAGE.readthedocs.io/en/latest
```

Comment:
```suggestion
    Please use PACKAGE.readthedocs.io not PACKAGE.readthedocs.io/en/latest
```

The message should probably also list the "location", e.g., "Documentation URL uses language-specific link"